src/write_organ_tags.py

```diff
@@ -6,9 +6,77 @@
 #  Output will be used to filter and/or boost search results in Cell Annotation Platform (CAP)
 
 import argparse
-import sys
 import ruamel.yaml
 from SPARQLWrapper import SPARQLWrapper, JSON
+from collections import namedtuple
+
+
```

Comment:
I think most of these functions are clear to me. However, could you add 2-3 sentences of explanatory comments for each?

You could put these in `# comments` or in 
```
"""
comments
"""
```

Reply:
Well formatted docstrings for each function would be best.  I'd prefer these not to be treated as comments, but to describe args, output and what the function does.  Code comments are great, but better not to put them in docstrings.